src/main/java/uk/gov/hmcts/cmc/claimstore/services/staff/content/countycourtjudgment/AmountContentProvider.java

```diff
@@ -27,9 +29,10 @@ public AmountContent create(Claim claim) {
         BigDecimal claimAmount = ((AmountBreakDown) claim.getClaimData().getAmount()).getTotalAmount();
 
         BigDecimal admittedAmount = null;
-        if (claim.getResponse().isPresent()
-            && claim.getResponse().get() instanceof PartAdmissionResponse) {
-            admittedAmount = ((PartAdmissionResponse) claim.getResponse().get()).getAmount();
+        Response response = claim.getResponse().orElse(null);
+
+        if (response instanceof PartAdmissionResponse) {
```

Comment:
What if response is null in the case of CCJ by default? 'ContentProvider` class uses this method which is also used for generating PDF for default and admission.